scripts/.check-license.ignore

```diff
@@ -10,6 +10,9 @@
 .*\.sln$
 .*\.vsdx$
 
+# Directories with files that don't support embedding license info
+.*\corpus
```

Comment:
I don't think the `\` is correct.  This is a regex so don't need to escape the letter c.

Reply:
RE: `\`, yes, typo. 
RE: regex, yes, it should work on the entire directory, as it did in merging a `ebpf-for-windows\tests\libfuzzer\netebpfext_fuzzer\corpus\xdp` conflict (pushed forcibly on main) from a local branch of mine.